_snippets/code/tournament-notes.md

```diff
@@ -0,0 +1,2 @@
+!!! note "In version 1.9.0 we updated our templating language for expressions"
+    Should you run into any issues working with expressions on 1.9.0, we recommend switching back N8N_EXPRESSION_EVALUATOR to tmpl and reporting this issue on our [forums](https://community.n8n.io/) so we can review and fix.
```

Comment:
```suggestion
   If you have issues with expressions in 1.9.0:
   
   * Please report the issue on the [forums](https://community.n8n.io/){:target=_blank .external-link}.
   * Self-hosted users can switch back to RiotTmpl: set `N8N_EXPRESSION_EVALUATOR` to `tmpl`.  Refer to [Environment variables](/hosting/environment-variables/) for more information on configuring self-hosted n8n.  
```

Reply:
Explanation:

- We don't use "we" language: n8n is referred to in the third person
- Some rephrasing for simplicity
- Make sure the forum link opens in a new tab (we do this for all external links)
- Make clear only self-hosted users can revert, and guide them to the config docs
- Code highlight environment variables and their settings